input/pages/app-launch.md

```diff
@@ -335,13 +335,13 @@ The app supplies the following parameters to the EHR’s "authorize" endpoint.
       <td><span class="label label-success">required</span></td>
       <td>
 
-Must describe the access that the app needs, including scopes like
+Must describe the access that the app needs including scopes like
```

Comment:
```suggestion
Must describe the access that the app needs, including scopes like
```

I think the `,` improves readability by separating the descriptive clause

Reply:
I don't, but its not a big deal either way to me.